bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html

```diff
@@ -0,0 +1,96 @@
+{% elif LANG == "es-ES" %}
+  {% set main_title = 'Di “Hello” a Firefox translations (edición de viaje)' %}
+  {% set main_tagline = 'Firefox translations ya está disponible para Android.' %}
+  {% set main_body = 'Para que puedas traducir la web de tu tour de snorkel directamente desde Firefox… y desde la playa. ¡Glug!' %}
+  {% set cta = 'Escanea el código QR' %}
+  {% set qr_code_url = 'img/l10n/es-ES/firefox/wnp-129/qr-code-es.png' %}
+{% elif LANG == "it" %}
+  {% set main_title = 'Hola! Sono arrivate le traduzioni di Firefox (anche per mobile)' %}
+  {% set main_tagline = 'Firefox translations è disponibile su Android!' %}
+  {% set main_body = 'Ora puoi tradurre il sito delle immersioni subacquee che farai quest’estate, direttamente da Firefox e pure dalla spiaggia. Plufff!' %}
+  {% set cta = 'Scannerizza il QR-Code' %}
+  {% set qr_code_url = 'img/l10n/it/firefox/wnp-129/qr-code-it.png' %}
+{% elif LANG == "pl" %}
+  {% set main_title = 'Hola! Przywitaj się z translatorem Firefoksa (wersja plażowa)' %}
+  {% set main_tagline = 'Funkcja Firefox translations jest teraz dostępna dla Androida!' %}
+  {% set main_body = 'Możesz w  spokoju tłumaczyć stronki wycieczek, na które pojedziesz w te wakacje bezpośrednio w Firefoksie. Plum plum!' %}
+  {% set cta = 'Zeskanuj kod QR' %}
+  {% set qr_code_url = 'img/l10n/pl/firefox/wnp-129/qr-code-pl.png' %}
+{% else %}
+  {% set main_title = 'Say Hola to Firefox translations (Beach Edition)' %}
+  {% set main_tagline = 'Firefox translations is now on Android!' %}
+  {% set main_body = 'So you can translate the website for snorkeling tours directly at location in Firefox. Blub!' %}
+  {% set cta = 'Scan the QR-Code' %}
+  {% set qr_code_url = 'img/l10n/en-GB/firefox/wnp-129/qr-code-uk.png' %}
+{% endif %}
+
+{% block site_header %}{% endblock %}
+{% block wnp_content %}
+  <section class="wnp-content-main">
+    <div class="mzp-c-split mzp-l-split-body-wide mzp-l-content mzp-t-content-xl">
+      <div class="mzp-c-split-container">
+        <div class="mzp-c-split-body ">
+          {{ picture(
+            url='img/firefox/whatsnew/whatsnew129-eu/translation-icon.svg',
+            optional_attributes={
+              'class': 'wnp-main-image',
+              'width': '50',
+              'height': '50',
+            }
+          ) }}
+          <h1 class="mzp-u-title-md">{{ main_title }}</h1>
+          <p class="wnp-sub-title"><strong>{{ main_tagline }}</strong></p>
+          <p>{{ main_body }}</p>
+        </div>
+        <div class="mzp-c-split-media mzp-c-emphasis-box">
+          <div class="wnp-cta-container">
+            <p class="wnp-cta-image"></p>
+            <p class="wnp-cta-content"><strong>{{ cta }}</strong></p>
+            {{ picture(
+              url=qr_code_url,
+              optional_attributes={
+                'class': 'wnp-main-image',
+                'width': '150',
+                'height': '150',
+              }
+            ) }}
```

Comment:
FYI, the localised images in img/l10n, when all named the same, just in different locale folders, can be referenced by the locale automagically via https://bedrock.readthedocs.io/en/latest/coding.html#l10n-img (or in this case ` "l10n": True,` on the `picture` helper as shown at the end of https://bedrock.readthedocs.io/en/latest/coding.html#resp-img …)

Reply:
Thanks for the suggestions!

Reply:
👍 yea I was kinda surprised when doing l10n images the first time, somewhat expecting things to end up elsewhere than was the reality;D so thought to share that early on to point out the bits in docs that tackle this;)